Simulator_engine/Farm.py

```diff
@@ -21,7 +21,8 @@ def __init__(self, delta_time=0.5, fish_count=200_000, L_0=2000, name=None, init
                  treat_automatic_eff = None, treat_automatic_thres = None,
                  fish_count_history=None,
                  temperature=None, mean_temprature=10,
-                 CF_data=None, cleanEff=1, lice_mortality=[0.01, 0.01, 0.02, 0.02, 0.02, 0.02], 
+                 CF_data=None, cleanEff=1, CF_lice_min = 0.4,
```

Comment:
shall the default for `CF_lice_min` be 0.4 and not 0.0?
and make shure that it can't be less then 0